.changes/v2.20.0/536-bug-fixes.md

```diff
@@ -0,0 +1,2 @@
+* Fix a bug that prevented returning a specific error while authenticating with invalid password
```

Comment:
Just to be more specific:

```suggestion
* Fix a bug that prevented returning a specific error while authenticating provider with invalid password
```

Reply:
Oops, now that I reviewed the Terraform PR, this is more appropriate here in govcd:

```suggestion
* Fix a bug that prevented returning a specific error while authenticating client with invalid password
```

Reply:
Adjusted for the second comment.